pointsProgForm.php

```diff
@@ -114,11 +114,11 @@
                 </p>
                 <label for="name">* How many freezer meals would you like? </label>
                 <ul>
-                <li><input type="radio" id="freezer_meals" name="freezer_meals" value="2 Meals per month (Free)"> 2 Meals per month (Free)</li>
-                <li><input type="radio" id="freezer_meals" name="freezer_meals" value="2 Meals per month (Free)"> 4 meals per month (2 points)</li>
-                <li><input type="radio" id="freezer_meals" name="freezer_meals" value="2 Meals per month (Free)"> 6 meals per month (3 points)</li>
-                <li><input type="radio" id="freezer_meals" name="freezer_meals" value="2 Meals per month (Free)"> 8 meals per month (4 points)</li>
-                <li><input type="radio" id="freezer_meals" name="freezer_meals" value="2 Meals per month (Free)"> We do not want ANY freezer meals</li>
+                <li><input type="radio" id="freezer_meals" name="freezer_meals" value=2> 2 Meals per month (Free)</li>
```

Comment:
Maybe add a comment here explaining how the input is transferred into a variable here.

Reply:
After realizing my lack of comments throughout files, I plan to review all files and add comments as necessary.